certs/ca-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+data:
+  ca.crt: <ca.crt>
+kind: Secret
+metadata:
+  name: thanos-ca-secret
+  namespace: default
```

Comment:
Is the `namespace:` necessary here and in the other manifests?

Reply:
in the example file `multicluster-config/custom-workload-values-sample.yaml` we have defined path to this secret in annotation `nginx.ingress.kubernetes.io/auth-tls-secret:`, so I leave it there in case someone wants use values ​​from example file

Reply:
Ok and what about other manifests ?

Reply:
Other manifests have namespace set to monitoring, which is not mandatory. But for me it is better when when all things, which belongs to monitoring are in one namespace if its possible.

Reply:
Ok, and what about certs/server-secret.yaml ?

Reply:
That `namespace: default` is not necessary, it should be changed to monitoring or deleted